### PR TITLE
refactor: EpochTime as a pointer to allow for null value in JSON unmarshaling

### PR DESCRIPTION
**Pull request by sanderblue**

### CODE REVIEW

File: pkg/entities/types.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ var DashboardAlertSeverityTypes = struct {
 	WARNING: "WARNING",
 }
 
-// DashboardPermissions - Permission that represent visibility & editability
+// DashboardPermissions - Permisions that represent visibility & editability
```
Comment (sanderblue) [inline pkg/entities/types.go:52]: I thought we automatically fixed misspelled words 🤔

File: pkg/users/users_api.go [hunk 2]
```diff
@@ -14,7 +14,8 @@ func (a *Users) GetUser() (*User, error) {
 	return &resp.Actor.User, nil
 }
 
-const getUserQuery = `query { actor { user {
```
Comment (jthurman42) [inline pkg/users/users_api.go:17]: Are you using the latest version of tutone? This should not have changed 🤔

File: pkg/dashboards/types.go [hunk 3]
```diff
@@ -265,14 +265,6 @@ type DashboardWidgetNRQLQueryInput struct {
 	Query nrdb.NRQL `json:"query"`
 }
 
-// DashboardWidgetQueryInput -
```
Comment (jthurman42) [inline pkg/dashboards/types.go:268]: Where did this go?

Comment (sanderblue) [reply pkg/dashboards/types.go:268]: Honestly not sure 🤷  It's not referenced anywhere as far as I can tell.

File: .tutone.yml [hunk 4]
```diff
@@ -459,7 +459,7 @@ packages:
         field_type_override: nrtime.EpochSeconds
         skip_type_create: true
       - name: EpochMilliseconds
-        field_type_override: nrtime.EpochMilliseconds
+        field_type_override: "*nrtime.EpochMilliseconds"
```
Comment (jthurman42) [inline .tutone.yml:462]: 👍

File: pkg/dashboards/types.go (hunk 3 above)
Comment (jthurman42) [inline pkg/dashboards/types.go:268]: This was replaced with `DashboardWidgetNRQLQueryInput`, so can be removed.

**Review by zlesnr — APPROVED** [review]

File: pkg/users/users_api.go (hunk 2 above)
Comment (sanderblue) [reply pkg/users/users_api.go]: Should be good now.

File: pkg/entities/types.go [hunk 5]
```diff
@@ -2874,8 +2868,6 @@ func (x *DashboardEntityOutline) ImplementsEntityOutline() {}
 type DashboardLineWidgetConfiguration struct {
 	// nrql queries
 	NRQLQueries []DashboardWidgetNRQLQuery `json:"nrqlQueries,omitempty"`
-	// queries
```
Comment (jthurman42) [inline pkg/entities/types.go:2877]: This is the removal I was talking about, expected for all these config types

**Review by jthurman42 — APPROVED** [review]
:shipit:

**Pull request merged**